### PR TITLE
Added function for commands to add explicit help messages

### PR DESCRIPTION
**Pull request by lilggamegenius**
Allows commands to print their own help messages.

### CODE REVIEW

**Review by Zeni44 — CHANGES_REQUESTED** [review]
Indent it properly, Otherwise seems fine.

**Comment by lilggamegenius** [comment]
Fixed the Indents. Forgot to set editor to use actual tabs instead of 2 spaces.

File: lua/notagain/aowl/aowl.lua [hunk 1]
```diff
@@ -636,6 +646,12 @@ do -- commands
 	function aowl.GetHelpText(alias)
 		local command, msg = aowl.FindCommand(alias)
 		if not command then return false, msg end
+    
+    local str = awol.help[command]
+    
+    if str != nil then
+      return str
+    end
```
Comment (Zeni44) [inline lua/notagain/aowl/aowl.lua]: You forgot to indent line 649 - 654

**Review by Zeni44 — APPROVED** [review]

File: lua/notagain/aowl/aowl.lua [hunk 2]
```diff
@@ -636,6 +646,12 @@ do -- commands
 	function aowl.GetHelpText(alias)
 		local command, msg = aowl.FindCommand(alias)
 		if not command then return false, msg end
+    
+		local str = awol.help[command]
```
Comment (Techbot121) [inline lua/notagain/aowl/aowl.lua:650]: awol

**Pull request merged**